xchange-ftx/src/main/java/org/knowm/xchange/ftx/FtxAdapters.java

```diff
@@ -350,16 +352,17 @@ public static BigDecimal lendingRounding(BigDecimal value) {
     return value.setScale(4, RoundingMode.DOWN);
   }
 
-  public static Ticker adaptTicker(FtxResponse<FtxMarketDto> ftxMarketResp,
-                                   FtxResponse<List<FtxCandleDto>> ftxCandlesResp,
-                                   CurrencyPair currencyPair) {
+  public static Ticker adaptTicker(
+      FtxResponse<FtxMarketDto> ftxMarketResp,
+      FtxResponse<List<FtxCandleDto>> ftxCandlesResp,
+      CurrencyPair currencyPair) {
 
-    System.out.println("marketResp"  + ftxMarketResp.toString());
+    System.out.println("marketResp" + ftxMarketResp.toString());
```

Comment:
can you change this to using the proper logger class

Reply:
Yes, i will change them, i don't remember writing them.